lib/bridge/MatrixHandler.js

```diff
@@ -345,7 +345,7 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
             ).createRoom({
                 options: {
                     name: ircChannel,
-                    visibility: "private"
+                    visibility: ircServer.getJoinRule()
```

Comment:
Join rule != visibility. Copy what we do on [alias creation](https://github.com/matrix-org/matrix-appservice-irc/blob/bdeebc0f81636037016adf6bd16224293d10cf66/lib/bridge/MatrixHandler.js#L1072):
```js
            let res = yield botIntent.createRoom({
                options: {
                    name: channelInfo.channel,
                    visibility: (
                        channelInfo.server.shouldPublishRooms() ? "public" : "private"
                    ),
                    creation_content: {
                        "m.federate": channelInfo.server.shouldFederate()
                    },
                    initial_state: [
                        {
                            type: "m.room.join_rules",
                            state_key: "",
                            content: {
                                join_rule: channelInfo.server.getJoinRule()
                            }
                        },
                        {
                            type: "m.room.history_visibility",
                            state_key: "",
                            content: {
                                history_visibility: "joined"
                            }
                        }
                    ]
                }
            });
```